scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -136,13 +136,15 @@ class ScanCode(config: ScannerConfiguration) : LocalScanner(config) {
 
     override fun bootstrap(): File {
         val archive = when {
-            // Use the .zip file despite it being slightly larger than the .tar.bz2 file here as the latter for some
+            // Use the .zip file despite it being slightly larger than the .tar.gz file here as the latter for some
             // reason does not complete to unpack on Windows.
-            OS.isWindows -> "scancode-toolkit-$scannerVersion.zip"
-            else -> "scancode-toolkit-$scannerVersion.tar.bz2"
+            OS.isWindows -> "v$scannerVersion.zip"
+            else -> "v$scannerVersion.tar.gz"
```

Comment:
Why do you have to switch from bz2 to gz?

Reply:
Because there is no `.bz2` offered by GitHub for the source code archive.